components/story-inquiry-page/story-inquiry-form/StoryInquiryForm.js

```diff
@@ -379,21 +380,22 @@ export default function StoryInquiryForm({
             })}
           </InputFieldWrapper>
         </ResourceLinksContainer>
-        <SquareButtonWrapper>
-          <SquareButton
-            buttonText="SUBMIT MY STORY"
+        <ButtonWrapper>
+          <SubmitButton
+            aria-label="Submission Button"
+            color={colors.WHITE}
```

Comment:
Same here, you probably want to add the aria stuff as attrs on the styled component.